src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -303,8 +304,7 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
           toolsOpen={!!activeDrawer}
           toolsWidth={activeDrawerSize}
           sideSplitPanel={
-            splitPanelPosition === 'side' &&
-            splitPanel && (
```

Comment:
Why is this removed from the condition?

Reply:
I think this was to make it consistent with the bottom orientation, because it seemed redundant with [this line](https://github.com/cloudscape-design/components/blob/main/src/app-layout/visual-refresh-toolbar/skeleton/index.tsx#L100). Is this condition doing more than that?

Reply:
Okay, consistency is a good reason. I also cannot recall reasons why this condition was added